src/Container.ts

```diff
@@ -131,6 +131,16 @@ export class Container {
         });
     }
 
+    /**
+     * Clears the container by removing all previously registered artifacts from it.
+     */
+    static clear () {
```

Comment:
btw what if we call this method `reset` instead? Clear sounds like we are clearing some database.... Also I need method in container to remove some previously registered services like `Container.reset(A, B, C)`. Maybe this can be done now or in the future.